lidarrmetadata/api.py

```diff
@@ -345,32 +354,17 @@ def search_album():
     limit = None if limit < 1 else limit
 
     search_providers = provider.get_providers_implementing(provider.AlbumNameSearchMixin)
-    album_art_providers = provider.get_providers_implementing(provider.AlbumArtworkMixin)[::-1]
-
+    
     if search_providers:
-        albums = search_providers[0].search_album_name(query, artist_name=artist_name, limit=limit)
-
-        for album in albums:
-            album['Artists'] = [
-                get_artist_info(album['ArtistId'],
-                                include_albums=False,
-                                primary_types=None,
-                                secondary_types=None,
-                                release_statuses=None,
-                                check_blacklist=False)
-                ]
+        album_ids = search_providers[0].search_album_name(query, artist_name=artist_name, limit=limit)
+        
+        albums = [get_release_group_info(id['Id']) for id in album_ids]
```

Comment:
`id` is the name of a built-in function, so it's a convention to call such a thing `id_`.